run-tests.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+sudo docker compose exec accounts uv run manage.py test $1
```

Comment:
ex/ `./run-tests.sh src.thunderbird_accounts.authentication.tests.SignUpViewTestcase.test_not_on_allowed_list`

I've switched to sublime-text with lsp-pyright and lsp-ruff, so no more fancy pycharm test runners for me 😅